frontend/src/views/web/challenge/challenge-page.html

```diff
@@ -42,6 +42,23 @@
                                 </a>
                             </span>
                         </span>
+
+                        <br>
+                        <span>
+                            <a class="pointer" ng-click="challenge.challengeDateDialog($event)">
+                                <strong class="text-light-black">Starts on:</strong>
+                                {{ challenge.page.start_date | date:'medium' }}
+                                <i class="fa fa-pencil" aria-hidden="true"></i>
```

Comment:
@takitsuse Minor change: Can we give a little more margin before the pencil icon like we have before `edit challenge title`?